src/main/java/org/gradlex/buildparameters/PluginCodeGeneration.java

```diff
@@ -29,7 +29,9 @@
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Iterator;
```

Comment:
unused

Reply:
? Why not remove it then?

Reply:
Ran out of time yesterday and added this comment as a reminder 😇